Source/UrlRequest_Win32.cpp

```diff
@@ -68,36 +68,41 @@ namespace UrlLib
             case UrlResponseType::String:
             {
                 return arcana::make_task(arcana::threadpool_scheduler, m_cancellationSource, [this, path] {
-                    std::ifstream file(path);
-                    if (!file.good())
+                    try
                     {
-                        std::stringstream msg;
-                        msg << "Failed to read file " << path.c_str();
```

Comment:
This is the main thing that was broken. This will result in a pointer being converted to a hex string.